engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -1342,6 +1343,19 @@ class demon_hunter_action_t : public Base
     ab::apply_affecting_aura( p->talent.havoc.any_means_necessary );
     ab::apply_affecting_aura( p->talent.havoc.dancing_with_fate );
 
+    ab::apply_affecting_aura( p->talent.vengeance.perfectly_balanced_glaive );
+    ab::apply_affecting_aura( p->talent.vengeance.meteoric_strikes );
+    ab::apply_affecting_aura( p->talent.vengeance.shear_fury );
+    ab::apply_affecting_aura( p->talent.vengeance.roaring_fire );
+    ab::apply_affecting_aura( p->talent.vengeance.agonizing_flames );
+    ab::apply_affecting_aura( p->talent.vengeance.extended_spikes );
+    ab::apply_affecting_aura( p->talent.vengeance.burning_blood );
+    ab::apply_affecting_aura( p->talent.vengeance.ruinous_bulwark );
+    ab::apply_affecting_aura( p->talent.vengeance.volatile_flameblood );
```

Comment:
Volatile Flameblood will need to be implemented manually as a gain in the Immolation Aura impact (is it per hit? or per execute?)

Look at Isolated Prey handling in Fel Rush as an example.

Reply:
I'll pull that out into a separate PR then